libs/platform/src/lib/components/info-label/info-label.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, ElementRef, ViewEncapsulation } from '@angular/core';
+import { InfoLabelComponent, CssClassBuilder, applyCssClass } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-info-label',
+    templateUrl: './info-label.component.html',
+    encapsulation: ViewEncapsulation.None
+})
+export class InfoLabelPlatformComponent extends InfoLabelComponent implements CssClassBuilder {
```

Comment:
What is the added value  ? How we are enhancing this component ?

Reply:
Conversion of directive to Component